### DISCUSSION TITLE
multi commands in `scritps`

### DISCUSSION THREAD

**Discussion by hvlxh** [npm/feedback#919] [Polls]
wrong, ugly code
```json
"scripts": {
  "run": "a & b & c"
}
```

correct, better code
```json
"scripts": {
  "run": [
    "a",
    "b",
    "c"
  ]
}
```

**Comment by ljharb** [comment]
See:
 - https://github.com/npm/rfcs/issues/190
 - https://github.com/npm/rfcs/issues/610
 - https://github.com/npm/rfcs/issues/573
 - https://github.com/npm/feedback/discussions/781
